vendor/bat-native-ads/src/bat/ads/internal/tokens/redeem_unblinded_token/user_data/confirmation_dto_user_data_builder.cc

```diff
@@ -36,8 +36,8 @@ void Build(const std::string& creative_instance_id,
   const base::DictionaryValue locale_user_data = GetLocale();
   user_data.MergeDictionary(&locale_user_data);
 
-  const base::DictionaryValue experiment_user_data = GetExperiment();
-  user_data.MergeDictionary(&experiment_user_data);
+  const base::DictionaryValue studies_user_data = GetStudies();
```

Comment:
nit: `studies` instead of `studies_user_data`

Reply:
I was following "convention" with previous lines `platform_user_data`, `build_channel_user_data`, and so on. Should we rename all the above?